test/functional/neutronless/loadbalancer/test_loadbalancer_disconnected.py

```diff
@@ -0,0 +1,244 @@
+# coding=utf-8
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+from ..testlib.service_reader import LoadbalancerReader
+from copy import deepcopy
+import json
+import logging
+import os
+import pytest
+import requests
+
+
+from f5_openstack_agent.lbaasv2.drivers.bigip.resource_helper import \
+    ResourceType
+
+requests.packages.urllib3.disable_warnings()
+
+LOG = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def services():
+    neutron_services_filename = (
+        os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                     '../../testdata/service_requests/create_delete_disconnected_lb.json')
+    )
+    return (json.load(open(neutron_services_filename)))
+
+@pytest.fixture
+def disconnected_service_no_seg():
+    neutron_services_filename = (
+        os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                     '../../testdata/service_requests/create_delete_disconnected_noseg_lb.json')
+    )
+    return (json.load(open(neutron_services_filename)))
+
+@pytest.fixture(scope="module")
+def icd_config():
+    oslo_config_filename = (
+        os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                     '../../config/basic_agent_config.json')
+    )
+    OSLO_CONFIGS = json.load(open(oslo_config_filename))
+
+    config = deepcopy(OSLO_CONFIGS)
+    config['icontrol_hostname'] = pytest.symbols.bigip_mgmt_ip_public
+    config['icontrol_username'] = pytest.symbols.bigip_username
+    config['icontrol_password'] = pytest.symbols.bigip_password
+    config['f5_vtep_selfip_name'] = pytest.symbols.f5_vtep_selfip_name
+
+    return config
+
+
+def test_featureoff_nosegig_lb(bigip, disconnected_service_no_seg,
+                               icd_config, icontrol_driver):
+
+    service_iter = iter(disconnected_service_no_seg)
+    service = service_iter.next()
+    lb_reader = LoadbalancerReader(service)
+    env_prefix = icd_config['environment_prefix']
+    fake_rpc = icontrol_driver.plugin_rpc
+    hostname = pytest.symbols.bigip_hostname
+    icd_config['f5_network_segment_physical_network'] = None
+
+    folder = '%s_%s' % (env_prefix, lb_reader.tenant_id())
+    # Make sure we are starting clean.
+    assert not bigip.folder_exists(folder)
+
+    # Create the loadbalancer (should not complete)
+    lb_pending = icontrol_driver._common_service_handler(service)
+    assert not lb_pending
```

Comment:
If the feature is off and there is no seg-id, shouldn't common service handler raise?  I wouldn't expect it to return False since this is a persistent fatal error.